types/types.proto

```diff
@@ -356,6 +356,21 @@ message PodRemoveResponse {
   string cause  = 2;
 }
 
+message ContainerLogsRequest {
+  string  container =1;
+  bool    follow    =2;
+  bool    timestamps=3;
```

Comment:
comparing to existing definition, `bool    timestamps = 3;` seems better.
there is one blank at least before `=` and one blank exactly after `=`.